test/lit/forward-declared-types.wast

```diff
@@ -2,10 +2,10 @@
 
 ;; RUN: wasm-opt %s -all -S -o - | filecheck %s
 
-;; CHECK: (type $none_=>_none (func))
-;; CHECK: (type $none_=>_ref?|{ref?|[rtt_2_$none_=>_none]|_ref?|none_->_none|}| (func (result (ref null ${ref?|[rtt_2_$none_=>_none]|_ref?|none_->_none|}))))
-;; CHECK: (type ${ref?|[rtt_2_$none_=>_none]|_ref?|none_->_none|} (struct (field (ref null $[rtt_2_$none_=>_none])) (field (ref null $none_=>_none))))
-;; CHECK: (type $[rtt_2_$none_=>_none] (array (rtt 2 $none_=>_none)))
+;; CHECK: (type $func (func))
+;; CHECK: (type $none_=>_ref?|{ref?|[rtt_2_$func]|_ref?|none_->_none|}| (func (result (ref null $struct))))
```

Comment:
Would it be possible to change the type name generation so that this becomes `(type $none_=>_ref?|$struct| ...)`? That would be much more readable.

Reply:
Great point! Definitely much better that way. Fixed.